### ISSUE TITLE
Yet another composer is slow A.F. bug

### ISSUE THREAD

**Issue by kwhat** [composer/composer#8464]
I am aware that this has been reported about a 1000 times but I am going to report it again because I am unsatisfied with the it must be your environment or packagest is slow today answers.  This is destroying my productivity.  I am using WSLv1 + Ubuntu with a pretty generic Laravel composer.json.  I have tried with and without Prestissimo.  Also disabled xdebug just to be sure.

`composer -vvv --profile require twilio/sdk`

This portion took several minutes to complete, and we haven't even started profiling yet.
```
Reading ./composer.json
Loading config file ./composer.json
Checked CA file /etc/ssl/certs/ca-certificates.crt: valid
Executing command (/mnt/c/Users/admin/www/HE-Matrix): git branch --no-color --no-abbrev -v
Reading /home/ubuntu/.composer/composer.json
Loading config file /home/ubuntu/.composer/composer.json
Reading /mnt/c/Users/admin/www/HE-Matrix/vendor/composer/installed.json
Reading /home/ubuntu/.composer/vendor/composer/installed.json
Loading plugin Hirak\Prestissimo\Plugin
Downloading https://packagist.org/packages.json
Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/packages.json into cache
    1/7:        http://packagist.org/p/provider-latest$e7a621d3f40712230328c843f16e97c9b88bad55dfec23a1ee4440e2acae9b07.json
    2/7:        http://packagist.org/p/provider-2019-01$3f382393014fd22597eaaf47c3ea0d92df44219dccc3c0d58191893dec1d866a.json
    3/7:        http://packagist.org/p/provider-2019-10$1267be661d6ab7bf7c6aa480636e4288145b4a8cdca0d3bf913a09ffd5e72856.json
    4/7:        http://packagist.org/p/provider-2019-04$22ae3be2a552d14196e3b4a3129c2012416bf244aded66f36ecf66312c6a1d98.json
    5/7:        http://packagist.org/p/provider-2016$3d6fdd8c8c5b8b4db22c241532fec4bb10b489eff4b3abc2a138ee52f20899c3.json
    6/7:        http://packagist.org/p/provider-2019-07$826ccf03fa6b20fbc4af44ba2daf55607c96a8a67041fa419459064345a68e3a.json
    7/7:        http://packagist.org/p/provider-2018$345c8c22aa7866ed1cb2cdde32c422c773fd1c026c9058ad704cee942934c14d.json
    Finished: success: 7, skipped: 0, failure: 0, total: 7
Running 1.6.3 (2018-01-31 16:28:17) with PHP 7.2.24-0ubuntu0.18.04.1 on Linux / 4.4.0-18362-Microsoft
```

What is going on here? 50s, 168s, 64s, 144s, 329s.... we are definitely not downloading anything at these steps.
```
[8.3MB/1.45s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/packages.json into cache
[8.3MB/1.45s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/p-provider-2013.json from cache
[11.9MB/1.47s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/p-provider-2014.json from cache
[19.6MB/1.50s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/p-provider-2015.json from cache
[51.0MB/51.77s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/p-provider-2016.json into cache
[52.2MB/134.23s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/p-provider-2017.json from cache
[104.7MB/302.45s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/p-provider-2018.json into cache
[113.4MB/327.79s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/p-provider-2019-01.json into cache
[125.4MB/357.06s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/p-provider-2019-04.json into cache
[138.7MB/393.26s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/p-provider-2019-07.json into cache
[152.1MB/424.28s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/p-provider-2019-10.json into cache
[151.0MB/424.34s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/p-provider-archived.json from cache
[156.0MB/441.70s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/p-provider-latest.json into cache
[155.7MB/442.13s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-twilio$sdk.json into cache
[155.6MB/442.14s] Using version ^5.39 for twilio/sdk
[155.6MB/442.14s] ./composer.json has been updated
[155.9MB/442.15s] Reading ./composer.json
[155.9MB/442.15s] Loading config file ./composer.json
[155.9MB/442.15s] Executing command (/mnt/c/Users/admin/www/HE-Matrix): git branch --no-color --no-abbrev -v
[156.2MB/442.23s] Reading /home/ubuntu/.composer/composer.json
[156.2MB/442.23s] Loading config file /home/ubuntu/.composer/composer.json
[156.2MB/442.23s] Reading /mnt/c/Users/admin/www/HE-Matrix/vendor/composer/installed.json
[156.5MB/442.29s] Reading /home/ubuntu/.composer/vendor/composer/installed.json
[156.6MB/442.29s] Loading plugin Hirak\Prestissimo\Plugin_composer_tmp0
[156.6MB/442.29s] Downloading https://packagist.org/packages.json
[156.6MB/442.73s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/packages.json into cache
[156.6MB/456.02s]     1/4:      http://packagist.org/p/provider-latest$ae8cd2a4b24417c9e40e0392c0d1a4efa906360a759a6e50f4c5ba3ea902440a.json
[156.6MB/471.65s]     2/4:      http://packagist.org/p/provider-2019-04$11aea915654649c3cf054b27cfb02693e6c2a7940ebdd18f2a0b9c84559e9d91.json
[156.6MB/472.13s]     3/4:      http://packagist.org/p/provider-2019-10$6dfe3ca401dd195f9a14c41cccb991c2ef3d4656dd71523262ab930fc6816d84.json
[156.6MB/475.54s]     4/4:      http://packagist.org/p/provider-2019-07$5fcb0441bfa82d3cfb82fbecfafa72dddaed2daef0d358aef8e6810bbbde5416.json
[156.6MB/476.13s]     Finished: success: 4, skipped: 0, failure: 0, total: 4
[156.7MB/476.16s] Reading ./composer.lock
[157.8MB/476.17s] Loading composer repositories with package information
[158.0MB/476.76s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/packages.json into cache
[158.0MB/476.76s] Updating dependencies (including require-dev)
[158.8MB/476.77s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/p-provider-2013.json from cache
[162.4MB/476.79s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/p-provider-2014.json from cache
[170.1MB/476.82s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/p-provider-2015.json from cache
[182.3MB/476.87s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/p-provider-2016.json from cache
[200.2MB/476.94s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/p-provider-2017.json from cache
[221.7MB/477.03s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/p-provider-2018.json from cache
[252.9MB/477.14s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/p-provider-2019-01.json from cache
[276.2MB/510.93s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/p-provider-2019-04.json into cache
[273.5MB/574.55s] Downloading http://packagist.org/p/provider-2019-07%245fcb0441bfa82d3cfb82fbecfafa72dddaed2daef0d358aef8e6810bbbde5416.json
[273.9MB/811.08s] Content-Length mismatch, received 193915 out of 1275520 bytes: (H4sIAAAAAA...
[274.0MB/868.25s] Downloading http://packagist.org/p/provider-2019-07%245fcb0441bfa82d3cfb82fbecfafa72dddaed2daef0d358aef8e6810bbbde5416.json
[290.4MB/1012.37s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/p-provider-2019-07.json into cache
[302.9MB/1038.28s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/p-provider-2019-10.json into cache
[301.8MB/1038.30s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/p-provider-archived.json from cache
[306.9MB/1052.61s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/p-provider-latest.json into cache
[306.2MB/1052.63s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-dnoegel$php-xdg-base-dir.json from cache
[306.2MB/1052.64s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-doctrine$inflector.json from cache
[306.4MB/1052.64s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-doctrine$instantiator.json from cache
[306.5MB/1052.64s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-doctrine$lexer.json from cache
[306.6MB/1052.64s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-dragonmantank$cron-expression.json from cache
[306.7MB/1052.64s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-egulias$email-validator.json from cache
[307.0MB/1052.65s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-erusev$parsedown.json from cache
[307.4MB/1052.65s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-psr$log.json from cache
[307.6MB/1052.65s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-filp$whoops.json from cache
[308.0MB/1052.65s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-firebase$php-jwt.json from cache
[308.2MB/1052.66s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-guzzlehttp$guzzle.json from cache
[309.3MB/1052.66s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-guzzlehttp$psr7.json from cache
[309.5MB/1052.67s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-psr$cache.json from cache
[309.8MB/1052.67s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-psr$http-message.json from cache
[310.3MB/1052.68s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-google$auth.json from cache
[310.5MB/1052.68s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-guzzlehttp$ringphp.json from cache
[310.6MB/1052.68s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-react$promise.json from cache
[310.8MB/1052.68s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-guzzlehttp$promises.json from cache
[310.9MB/1052.68s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-ralouphie$getallheaders.json from cache
[311.0MB/1052.69s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-guzzlehttp$streams.json from cache
[311.0MB/1052.69s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-monolog$monolog.json from cache
[311.6MB/1052.69s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-phpdocumentor$reflection-docblock.json from cache
[325.7MB/1053.84s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$serializer.json into cache
[328.9MB/1053.90s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-googleads$googleads-php-lib.json from cache
[329.5MB/1053.90s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-phpdocumentor$reflection-common.json from cache
[329.6MB/1053.90s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-phpdocumentor$type-resolver.json from cache
[329.8MB/1053.90s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-webmozart$assert.json from cache
[330.0MB/1053.91s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$polyfill-php55.json from cache
[342.4MB/1057.27s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$polyfill-ctype.json into cache
[345.5MB/1057.33s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-ircmaxell$password-compat.json from cache
[345.5MB/1057.33s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-hamcrest$hamcrest-php.json from cache
[345.5MB/1057.33s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-jakub-onderka$php-console-color.json from cache
[345.6MB/1057.33s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-jakub-onderka$php-console-highlighter.json from cache
[345.6MB/1057.33s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-justinrainbow$json-schema.json from cache
[346.0MB/1057.34s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-pda$pheanstalk.json from cache
[346.1MB/1057.34s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-pimple$pimple.json from cache
[346.2MB/1057.34s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-ulrichsg$getopt-php.json from cache
[346.5MB/1057.34s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-kwhat$legume.json from cache
[346.5MB/1057.34s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-psr$container.json from cache
[346.7MB/1057.35s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-league$flysystem.json from cache
[347.9MB/1057.36s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-nesbot$carbon.json from cache
[349.3MB/1057.36s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-opis$closure.json from cache
[349.6MB/1057.37s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-psr$simple-cache.json from cache
[349.7MB/1057.37s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-ramsey$uuid.json from cache
[350.2MB/1057.37s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-swiftmailer$swiftmailer.json from cache
[363.3MB/1060.36s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$console.json into cache
[365.2MB/1062.02s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$debug.json into cache
[369.2MB/1064.74s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$finder.json into cache
[372.4MB/1071.51s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$http-foundation.json into cache
[377.0MB/1075.24s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$http-kernel.json into cache
[380.2MB/1077.63s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$process.json into cache
[384.4MB/1079.42s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$routing.json into cache
[384.8MB/1080.40s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$var-dumper.json into cache
[378.0MB/1080.41s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-tijsverkoyen$css-to-inline-styles.json from cache
[377.0MB/1080.54s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-vlucas$phpdotenv.json into cache
[387.1MB/1081.55s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-laravel$framework.json into cache
[403.9MB/1083.35s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$translation.json into cache
[395.3MB/1083.38s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-kylekatarnls$update-helper.json from cache
[406.4MB/1088.15s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-paragonie$random-compat.json into cache
[397.0MB/1088.16s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$polyfill-intl-idn.json from cache
[414.4MB/1093.15s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$polyfill-iconv.json into cache
[404.0MB/1093.17s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$polyfill-mbstring.json from cache
[404.4MB/1093.18s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$contracts.json from cache
[404.6MB/1093.18s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$polyfill-php73.json from cache
[404.9MB/1093.19s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$service-contracts.json from cache
[403.6MB/1093.60s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$mime.json into cache
[417.4MB/1095.66s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$event-dispatcher.json into cache
[428.6MB/1100.72s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$polyfill-php56.json into cache
[409.6MB/1101.07s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$error-handler.json into cache
[409.7MB/1101.08s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$polyfill-php72.json from cache
[423.6MB/1103.36s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$css-selector.json into cache
[414.8MB/1103.37s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-phpoption$phpoption.json from cache
[414.9MB/1103.38s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$translation-contracts.json from cache
[414.4MB/1103.39s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$event-dispatcher-contracts.json from cache
[414.5MB/1103.39s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$polyfill-util.json from cache
[506.2MB/1122.70s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-illuminate$console.json into cache
[583.0MB/1192.08s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-illuminate$contracts.json into cache
[600.7MB/1252.97s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-illuminate$support.json into cache
[523.9MB/1253.01s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-psy$psysh.json from cache
[524.6MB/1253.02s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-laravel$tinker.json from cache
[525.7MB/1253.06s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-danielstjules$stringy.json from cache
[526.5MB/1253.06s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-nikic$php-parser.json from cache
[526.1MB/1253.11s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-mockery$mockery.json from cache
[526.4MB/1253.12s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-myclabs$deep-copy.json from cache
[526.5MB/1253.12s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-nunomaduro$collision.json from cache
[527.0MB/1253.12s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-phar-io$version.json from cache
[527.0MB/1253.12s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-phar-io$manifest.json from cache
[527.1MB/1253.12s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-sebastian$comparator.json from cache
[527.2MB/1253.12s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-sebastian$recursion-context.json from cache
[527.3MB/1253.13s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-phpspec$prophecy.json from cache
[527.5MB/1253.13s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-dflydev$markdown.json from cache
[527.6MB/1253.13s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-sebastian$diff.json from cache
[527.7MB/1253.13s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-sebastian$exporter.json from cache
[527.8MB/1253.13s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-phpunit$php-file-iterator.json from cache
[527.9MB/1253.13s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-phpunit$php-text-template.json from cache
[527.9MB/1253.14s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-phpunit$php-token-stream.json from cache
[528.2MB/1253.14s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-sebastian$code-unit-reverse-lookup.json from cache
[528.2MB/1253.14s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-sebastian$environment.json from cache
[528.4MB/1253.14s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-sebastian$version.json from cache
[528.4MB/1253.14s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-theseer$tokenizer.json from cache
[528.5MB/1253.14s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-phpunit$php-code-coverage.json from cache
[529.6MB/1253.15s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-phpunit$php-timer.json from cache
[529.6MB/1253.15s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-sebastian$global-state.json from cache
[529.7MB/1253.15s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-sebastian$object-enumerator.json from cache
[529.8MB/1253.15s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-sebastian$resource-operations.json from cache
[529.8MB/1253.16s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-phpunit$phpunit.json from cache
[534.8MB/1253.18s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-sebastian$object-reflector.json from cache
[534.7MB/1253.19s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-squizlabs$php-codesniffer.json from cache
[535.7MB/1253.19s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-laravel$laravel.json from cache
[536.3MB/1253.19s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-fideloper$proxy.json from cache
[536.5MB/1253.22s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-voku$portable-ascii.json from cache
[536.6MB/1253.24s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-twilio$sdk.json from cache
[537.7MB/1253.25s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-phpunit$phpunit-mock-objects.json from cache
[536.1MB/1256.53s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$property-access.json into cache
[535.3MB/1259.20s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$property-info.json into cache
[545.7MB/1264.12s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$yaml.json into cache
[549.7MB/1267.23s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$dependency-injection.json into cache
[547.0MB/1267.33s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-vimeo$psalm.json from cache
[552.5MB/1267.35s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-cordoval$hamcrest-php.json from cache
[552.5MB/1267.35s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-davedevelopment$hamcrest-php.json from cache
[552.5MB/1267.35s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-kodova$hamcrest-php.json from cache
[552.7MB/1267.40s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-tightenco$collect.json from cache
[645.8MB/1286.46s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-illuminate$auth.json into cache
[624.2MB/1293.69s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-illuminate$broadcasting.json into cache
[633.0MB/1306.88s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-illuminate$bus.json into cache
[650.1MB/1320.03s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-illuminate$cache.json into cache
[652.0MB/1335.13s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-illuminate$config.json into cache
[653.7MB/1348.41s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-illuminate$container.json into cache
[654.8MB/1371.67s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-illuminate$cookie.json into cache
[657.0MB/1421.12s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-illuminate$database.json into cache
[662.5MB/1463.25s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-illuminate$encryption.json into cache
[571.9MB/1792.55s] Downloading http://packagist.org/p/illuminate/events%245e257ed2b661ae178086dc372447aba541b60985c54344e3814c272a52ac6a81.json
[666.0MB/1805.53s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-illuminate$events.json into cache
[666.7MB/1818.66s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-illuminate$filesystem.json into cache
[667.6MB/1833.49s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-illuminate$hashing.json into cache
[669.3MB/1851.14s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-illuminate$http.json into cache
[671.5MB/1866.76s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-illuminate$log.json into cache
[673.0MB/1883.43s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-illuminate$mail.json into cache
[627.5MB/1889.83s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-illuminate$notifications.json into cache
[675.9MB/1907.54s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-illuminate$pagination.json into cache
[661.9MB/1922.86s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-illuminate$pipeline.json into cache
[679.2MB/1943.91s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-illuminate$queue.json into cache
[680.8MB/1968.34s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-illuminate$redis.json into cache
[682.9MB/1993.49s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-illuminate$routing.json into cache
[684.9MB/2018.37s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-illuminate$session.json into cache
[686.5MB/2036.10s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-illuminate$translation.json into cache
[688.4MB/2055.24s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-illuminate$validation.json into cache
[690.1MB/2074.46s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-illuminate$view.json into cache
[614.1MB/2074.51s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-league$flysystem-sftp.json from cache
[617.5MB/2074.56s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-rhumsaa$uuid.json from cache
[607.0MB/2076.88s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$lock.json into cache
[627.2MB/2079.33s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$config.json into cache
[618.4MB/2079.35s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-twig$twig.json from cache
[631.8MB/2081.88s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$browser-kit.json into cache
[659.5MB/2083.76s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$twig-bundle.json into cache
[651.3MB/2083.79s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$cache-contracts.json from cache
[651.4MB/2083.79s] Reading /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$http-client-contracts.json from cache
[650.3MB/2084.33s] Writing /home/ubuntu/.composer/cache/repo/https---packagist.org/provider-symfony$mailer.json into cache
[796.6MB/2086.95s] Resolving dependencies through SAT
[798.3MB/2087.13s] Dependency resolution completed in 0.180 seconds
[798.6MB/2088.04s]     1/1:     https://codeload.github.com/twilio/twilio-php/legacy.zip/71432b4b4962cbed2fa01a8cf836ae5cea35e6f3
[798.6MB/2088.04s]     Finished: success: 1, skipped: 0, failure: 0, total: 1
[798.6MB/2088.04s] Analyzed 25815 packages to resolve dependencies
[798.6MB/2088.04s] Analyzed 433737 rules to resolve dependencies
[799.8MB/2088.05s] Resolving dependencies through SAT
[799.8MB/2088.05s] Dependency resolution completed in 0.001 seconds
[798.9MB/2088.05s] Package operations: 1 install, 0 updates, 0 removals
[798.9MB/2088.05s] Installs: twilio/sdk:5.39.0
[798.9MB/2088.06s]   - Installing twilio/sdk (5.39.0): [799.0MB/2088.06s] Reading /home/ubuntu/.composer/cache/files/twilio/sdk/54a39c758511ac3b9fd6a8149ad8c81cd1ae5ba3.zip from cache
[799.0MB/2088.09s] Loading from cache[799.0MB/2088.09s]
[799.0MB/2088.09s]  Extracting archive[799.0MB/2088.09s] Executing command (CWD): unzip -qq  '/mnt/c/Users/admin/www/HE-Matrix/vendor/twilio/sdk/b592ea750d69e625aa5e24f3db02a59e' -d '/mnt/c/Users/admin/www/HE-Matrix/vendor/composer/941e22dc'
[799.0MB/2097.15s]     REASON: Required by the root package: Install command rule (install twilio/sdk 5.39.0)
[799.0MB/2097.15s]
[608.6MB/2098.86s] Reading /mnt/c/Users/admin/www/HE-Matrix/vendor/composer/installed.json
[609.4MB/2098.87s] Executing command (/mnt/c/Users/admin/www/HE-Matrix/vendor/kwhat/legume): git log -n1 --pretty=%ct '0fc8abf01c8bd0c75ad27049de827620db38afa4'
[609.0MB/2098.99s] Writing lock file
[609.0MB/2098.99s] Generating optimized autoload files
[620.1MB/2103.98s] > post-autoload-dump: Illuminate\Foundation\ComposerScripts::postAutoloadDump
[624.5MB/2104.04s] > post-autoload-dump: @php artisan package:discover --ansi
[624.6MB/2104.04s] Executing command (CWD): '/usr/bin/php7.2' -d allow_url_fopen='1' -d disable_functions='' -d memory_limit='-1' artisan package:discover --ansi
[624.6MB/2104.79s] Discovered Package: laravel/tinker
Discovered Package: nesbot/carbon
Discovered Package: nunomaduro/collision
Package manifest generated successfully.
[620.4MB/2105.35s] Memory usage: 620.42MB (peak: 799.86MB), time: 2105.35s
```

2105.35s... Seriously, is this a joke? I cant imagine that composer require taking 35+ minutes to complete is the intended behavior.

**Comment by Seldaek** [comment]
We're working on this in v2, this is being addressed, just bear with us or use the 2.0 branch already if you wanna live on the bleeding edge :)

**Comment by kwhat** [comment]
Ok, np.  I kept running into this when reading from cache and the only solutions I came across were things like slow internet or other miss-configuration.  Just want to make sure this isn't like the FireFox 3 doesn't leaking memory problem and is on the radar as an actual problem.  Thanks for the update. 👍
